app/routes/upload.tsx

```diff
@@ -0,0 +1,21 @@
+import { Outlet } from '@remix-run/react'
+import Logo from '~/components/Logo'
+import Stepper from '~/components/Stepper'
+
+const upload = () => {
+  return (
+    <div className="max-w-screen-lg min-h-screen mx-auto">
+      <header className="w-full pt-16">
+        <Logo />
+      </header>
+
+      <div className="flex items-center justify-center flex-col">
+        <Stepper />
+
```

Comment:
**praise**: The integration of the Stepper component within the layout is seamless and enhances user navigation.